ansible_collections/arista/avd/molecule/eos_designs_unit_tests/intended/structured_configs/cv-pathfinder-transit.yml

```
@@ -136,21 +136,31 @@ management_api_http:
   - name: MGMT
   enable_https: true
 ethernet_interfaces:
-- name: Ethernet1
+- name: Ethernet1.42
   peer_type: l3_interface
   ip_address: dhcp
   shutdown: false
-  type: routed
+  type: l3dot1q
+  encapsulation_dot1q_vlan: '42'
```

Encapsulation should be an interger.
```suggestion
  encapsulation_dot1q_vlan: 42
```